MAUI/DataForm/Editors.md

```diff
@@ -0,0 +1,961 @@
+
+You can display the complex type property values in picker editor by using the [GetSource]`Link` override method of IDataFormSourceProvider class, which is used to get source list as complex property values for picker editor and set it to `IDataFormSourceProvider` property of SfDataForm. You need to use `GenerateDataFormItem`event to set [DisplayMemberPath]`Link` and [ValueMemberPath]`Link` property value DataFormPickerItem for complex type property.
+
+N> Class cannot be directly set as data type for picker editor in this complex type scenario.
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    dataForm.ItemsSourceProvider = new DataFormItemsSourceProvider();
+    dataForm.RegisterEditor("City", DataFormEditorType.Picker);
+    dataForm.DataObject = new DataModel();
+        dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+ 
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null && e.DataFormItem.FieldName == "City" && e.DataFormItem is DataFormPickerItem pickerItem)
+        {
+            pickerItem.DisplayMemberPath = "City";
+            pickerItem.ValueMemberPath = "PostalCode";
+        }
+    } 
+ 
+    public class DataFormItemsSourceProvider : IDataFormSourceProvider
+    {
+         public object GetSource(string sourceName)
+        {
+            if (sourceName == "City")
+            {
+                List<Address> details = new List<Address>();
+                details.Add(new Address() { City = "Chennai", PostalCode = 1 });
+                details.Add(new Address() { City = "Paris", PostalCode = 2 });
+                details.Add(new Address() { City = "Vatican", PostalCode = 3 });
+
+                return details;
+            }
+        return new List<string>();
+        }
+    }
+
+    public class DataModel
+    {
+        public String FirstName { get; set; } 
+        public string City { get; set; }
+    }
+
+    public class Address
+    {
+        public int PostalCode { get; set; }
+        public string City { get; set; }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+`Image`
+
+
+
+## RadioGroup editor
+
+In the [RadioGroup](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/radiobutton?view=net-maui-7.0) editor, the [RadioGroup](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/radiobutton?view=net-maui-7.0) control is loaded.
+
+The [items]`Link` for [RadioGroup](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/radiobutton?view=net-maui-7.0) is generated for `enum` and `List` data type properties. In order to add [RadioGroup](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/radiobutton?view=net-maui-7.0) editor in the DataForm, you need to register editor as [RadioGroup](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/radiobutton?view=net-maui-7.0) for the required property by using the [RegisterEditor]`Link` method.
```

Comment:
you need to register editor as [RadioGroup] => you need to register editor as [DataFrmEditorType.RadioGroup]

Remove link

Reply:
changed like above mentioned sentence